pkg/security/proto/api/api_grpc.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go-grpc. DO NOT EDIT.
 // versions:
 // - protoc-gen-go-grpc v1.2.0
-// - protoc             v3.12.4
+// - protoc             v3.6.1
```

Comment:
Why did this version go backwards?

Reply:
My version is probably just out of date :O good catch, I'll update that!